runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/components.json

```diff
@@ -31,6 +34,13 @@
     "Component.Tile.Process.Footer.Icon.Tooltip.Private": "Private process",
     "Component.Tile.Process.Footer.Icon.Tooltip.Public": "Public process",
     "Component.Tile.Process.Footer.Icon.Tooltip.Scheduled": "Scheduled",
+    "Component.Tile.Process.Footer.Status.Completed": "COMPLETED",
+    "Component.Tile.Process.Footer.Status.InProgress": "IN PROGRESS",
+    "Component.Tile.Process.Footer.Status.Initializing": "INITIALIZING",
+    "Component.Tile.Process.Footer.Status.Errored": "ERRORED",
+    "Component.Tile.Process.Footer.Status.Stopped": "STOPPED",
+    "Component.Tile.Process.Footer.Status.Terminated": "TERMINATED",
```

Comment:
isn't it already in the translations?

Reply:
if so (I think it is), then do not create another translation that covers the same element of the app. It will be hard to track when we would like to introduce a change

Reply:
it is in "Component.HistoryTable.Status" which points to different element

Reply:
then the translations from Component.HistoryTable.Status should be hoisted to more common file with translations, for instance process.json

Reply:
Done.